src/main/java/mops/gruppen2/domain/model/group/wrapper/Link.java

```diff
@@ -0,0 +1,21 @@
+package mops.gruppen2.domain.model.group.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.UUID;
+
+@Value
+public class Link {
+
+    @NotNull
+    @Size(min = 36, max = 36)
+    @JsonProperty("value")
+    String value;
+
+    public static Link RANDOM() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'RANDOM' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/tobi_2/gruppenfindung/pullRequest?prid=5356872)